ladders-matchmaker.js

```diff
@@ -75,14 +103,24 @@ class Matchmaker {
 			// User feedback for renames handled elsewhere.
 			return;
 		}
+
 		if (oldUserid !== user.userid) return;
 		if (validTeam === false) return;
 
 		const search = new Search(user.userid, validTeam, rating);
 		this.addSearch(search, user, formatid);
 	}
 
-	matchmakingOK(search1, search2, user1, user2, formatid) {
+	/**
+	 * Verifies whether or not a match made between two users is valid.
+	 * @param {Search} search1
+	 * @param {Search} search2
+	 * @param {?User} [user1 = null]
+	 * @param {?User} [user2 = null]
+	 * @param {string} format
+	 * @return {number | false | void}
+	 */
+	matchmakingOK(search1, search2, user1 = null, user2 = null, format) {
```

Comment:
Isn't there an alternative other than using defaults?
``user1`` and ``user2`` being falsey is exceptional (by which I mean *buggy* path). It's odd to make the docs more *extensive* for this case, as if it were supported.

Reply:
I suppose it's unnecessary, since if either of the two are undefined the crashlogger would complain about it